evidence/s0me0ne-unkn0wn/0005-20250319-promotion.md

```diff
@@ -0,0 +1,69 @@
+# Evidence-0005: Promotion to Rank II
+
+|                 |                      |
+| --------------- | ---------------------|
+| **Report Date** | 2025/03/19           |
+| **Submitted by**| s0me0ne-unkn0wn      |
+
+
+## Member details
+
+- Current rank: I
+- Polkadot address: 13WGadgNgqSjiGQvfhimw9pX26mvGdYQ6XgrjPANSEDRoGMt
+- Date of initial induction: 2024/04/22
+- Date of last report: 2024/12/17
+- Area(s) of Expertise/Interest: WebAssembly, PolkaVM, PVF execution, throughput benchmarking
+
+## Reporting period
+
+- Start date: 2024/12/18
+- End date: 2025/03/19
+
+## Evidence
+
+Dear fellows,
+
+According to the Manifesto, a rank II member should be a core part of the team, clearly demonstrate the knowledge of the protocol, communicate, share knowledge, become deeply familiar with a major area of the protocol, and should not be asking dumb questions. Well, I have to confess, I'm still doing the latter.
+
+Otherwise, before joining the Fellowship and during a year of being a rank I member, I developed solid familiarity with the PVF execution pipeline[^6161][^98][^3139][^1192], WebAssembly[^nok][^exec][^6918][^13811], and PolkaVM[^10], as a major area of the Polkadot protocol. I shared my research and knowledge[^sl][^1343] on the problem of execution determinism as much as I could. I communicated with most of you directly or indirectly, hopefully making me a core part of the team. I also filed RFC-135[^135] proposing a change to the protocol, which has been approved and implemented as a part of ongoing PolkaVM integration PR[^pvm].
```

Comment:
Would be nice if instead of just dumping links, you could go a little bit in detail on what you work was about. What you found out/achieved. Also align on the requirements of a rank II as written in the manifesto.

Reply:
Also please mention when you got ranked up to rank I

Reply:
> Also please mention when you got ranked up to rank I

The current rank is in the "Member details" in the header, is that what you mean?

Reply:
Ah no, I've read it wrong. I'll add it, thank you for pointing out.